tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -4058,7 +4076,15 @@ TEST_P(OpConverter_FP32_FP16_INT32_Test, ConvertStridedSlice) {
           /*shrink_axis_mask=*/0,
           /*expected_output_dims=*/{1, 1, 1, 2},
           /*expected_output=*/{1, 2},
+          Status::OK(),
+          Status::OK(),
+          /*partial_input_dims=*/{1, 1, -1, -1},
       },
+      // End mask is present on all dimensions. This should override the
+      // fact
+      // that we are not providing an end value. For dynamic shape, it
+      // tests
+      // that we can infer tensor size when "end mask" is present.
```

Comment:
Would you please re-flow the comment to avoid "partial line" in the middle of the comment block? We try to put as much words as possible to each comment line under the limit of 80 chars, except for the last line. 

Can we also s/is present/is provided/ for here and other places in the test?
Also s/we are not providing an end value/the end value is 0/